### ISSUE TITLE
GoPro Hero 9 open GoPro API connection issue

### ISSUE THREAD

**Issue by koloyyee** [gopro/OpenGoPro#136]
Hey guys I have been trying to use the open-gopro API, I was following the python tutorial, I am only able to connect for 2 seconds then I ran into this, do you know what I should do?

```python
open_gopro/gopro.py", line 660, in _write_characteristic_receive_notification
logger.trace( # type: ignore
AttributeError: 'Logger' object has no attribute 'trace'
```
also I rain into this issue 

``` python
│                                                                                                  │
│   385 │   │   │   version = self.ble_command.get_open_gopro_api_version().flatten                │
│   386 │   │   │   version_str = f"{version.major}.{version.minor}"                               │
│   387 │   │   │   if version_str != "2.0":                                                       │
│ ❱ 388 │   │   │   │   raise InvalidOpenGoProVersion(version)                                     │
│   389 │   │   │   logger.info(f"Using Open GoPro API version {version_str}")                     │
│   390 │   │   │                                                                                  │
│   391 │   │   │   # Establish Wifi connection if desired                                         │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
InvalidOpenGoProVersion: Container: 
    major = 1
    minor = 0 is not a valid Open GoPro API version.
```

Thanks in advance!

**Comment by tcamise-gpsw** [comment]
Hello. Which part of the tutorial are you at?

Also, it looks like you somehow ended up with a newer version of the Open-GoPro Python SDK then what is specified in the requirements.txt (0.7.1) in the tutorials. Did you follow [these steps](https://gopro.github.io/OpenGoPro/tutorials/python/connect-ble#tutorials-package) to install the package? It is imperative to use that specific version because the tutorials are not updated at the same cadence that the [Open GoPro Python SDK](https://pypi.org/project/open-gopro/) is updated.

**Comment by koloyyee** [comment]
Thanks for replying, I was following the Open GoPro Python SDK.
Because when I was following the [official tutorial](https://gopro.github.io/OpenGoPro/tutorials/python/connect-ble#tutorials-package) it only `INFO:root:Found 0 matching devices.` even though I connect to the GoPro 9 with my MacBook Air M1.

And I do notice for the BLE 1.0 mentioned about GoPro 9, is it one of the reasons? thank you again

**Comment by wemat** [comment]
hello i have the same error ('Logger' object has no attribute 'trace') and I installed the package (pip) and run the sample code bellow. I tried it on my macbook pro and on a raspberry pi with the same result.

<img width="681" alt="Bildschirmfoto 2022-04-09 um 21 05 51" src="https://user-images.githubusercontent.com/45120368/162588230-ec35bdea-8537-4c2e-86da-8a6f029184f6.png">

**Comment by tcamise-gpsw** [comment]
Regarding the logger.trace error: this has been fixed in open-gopro versions >= 0.9.1 which the tutorials are now using.

Regarding the Invalid Open GoPro Version error, this is expected as the open-gopro Python SDK only supports version 2.0. There is no reason to use 1.0 at this point as 2.0 is available on all cameras that support 1.0.

Therefore closing this as there are no open issues to handle.
